app/src/main/res/values-de-rDE/strings.xml

```diff
@@ -23,143 +23,218 @@
+    <string name="user_popup_timeout">Timeout</string>
+    <string name="user_popup_delete">Nachricht löschen</string>
+    <string name="user_popup_ban">Ban</string>
+    <string name="user_popup_unban">Unban</string>
+    <string name="change_roomstate">Chatmodi</string>
+    <string name="confirm_user_ban_title">Ban bestätigen</string>
+    <string name="confirm_user_ban_message">Willst du diesen Nutzer wirklich bannen?</string>
+    <string name="confirm_user_ban_positive_button">Ban</string>
+    <string name="confirm_user_timeout_title">Timeout bestätigen</string>
+    <string name="confirm_user_timeout_positive_button">Timeout</string>
+    <string name="confirm_user_delete_title">Löschung bestätigen</string>
+    <string name="confirm_user_delete_message">Willst du diese Nachricht wirklich löschen?</string>
+    <string name="confirm_user_delete_positive_button">Löschen</string>
+    <string name="confirm_user_roomstate_title">Chatmodi anpassen</string>
+    <string name="roomstate_emote">\"Emote only\"-Modus</string>
+    <string name="roomstate_subs">\"Subscriber only\"-Modus</string>
+    <string name="roomstate_slow">Langsamer Modus (slow mode)</string>
+    <string name="roomstate_r9k">R9K-Modus</string>
+    <string name="roomstate_follow">Followermodus</string>
+    <string name="seconds">Sekunden</string>
+    <string name="minutes">Minuten</string>
+    <string name="add_command">Befehl hinzufügen</string>
+    <string name="remove_command">Befehl entfernen</string>
+    <string name="command_trigger_hint">Auslöser</string>
+    <string name="command__hint">Befehl</string>
+    <string name="commands_title">Benutzerdefinierte Befehle</string>
+    <string name="user_popup_report">Melden</string>
+    <string name="tts_ignore_list_add_user">Nutzer hinzufügen</string>
+    <string name="tts_ignore_list_remove_user">Nutzer entfernen</string>
+    <string name="tts_ignore_list_user_hint">Nutzer</string>
+    <string name="recent_upload_deletion_link">Löschlink:\n%1$s</string>
+    <string name="recent_uploads_clear">Liste leeren</string>
+
+    <string name="preference_timestamp_title">Zeitstempel anzeigen</string>
+    <string name="preference_mention_format_title">Erwähnungsformat</string>
+    <string name="preference_mention_format_summary">Ändere das Erwähnungsformat beim Anklicken eines Nutzernamens</string>
+    <string name="preference_roomstate_title">Chatmodi anzeigen</string>
+    <string name="preference_roomstate_summary">Zeigt Informationen über den momentanen Chatmodus, wie \"Subscriber-Only\"</string>
+    <string name="preference_streaminfo_title">Streaminfo anzeigen</string>
+    <string name="preference_streaminfo_summary">Zeigt Live-Status und Zuschauerzahl</string>
+    <string name="preference_notification_title">Benachrichtigung bei Erwähnungen</string>
+    <string name="preference_notification_summary">Benachrichtigung für Erwähnung auch erzeugen, wenn die App im Hintergrund läuft.</string>
```

Comment:
I think we should remove the `auch`, because this is the only way to even create a notification in dankchat. So the `auch` feels wrong here. I guess for english we have `also` which translates more like `zusätzlich` in this context?

Reply:
changed w/ 858939a